packages/pwa/app/components/_app/index.jsx

```diff
@@ -102,15 +102,16 @@ const App = (props) => {
 
     const onLogoClick = () => {
         // Goto the home page.
-        history.push(homeUrlBuilder(HOME_HREF, targetLocale))
+        const path = homeUrlBuilder(HOME_HREF, targetLocale)
+        history.push(path)
 
         // Close the drawer.
         onClose()
     }
 
     const onCartClick = () => {
-        // Goto the home page.
-        history.push(`/${targetLocale}/cart`)
+        const path = buildPathWithUrlConfigParams('/cart', {locale: targetLocale})
+        history.push(path)
```

Comment:
Ideally, we should be able to use the hook `useNavigation`, which has been refactored to take url config into the account to handle navigation to another page. However, due to the limit that the App component does not access to the intl object, we have to use `history.push` here, so we have to use `buildPathWithUrlConfigParams` to rebuild our path before passing it to `history.push`

When the App component can access to intl object (which we are currently working on this sprint), we will need to refactor this piece of code to adapt to that change, and it should be simpler than this.